api/qr.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Do not move classes into the API structure, keep it simple as control structures that call classes
for example, move everything to QrCodeUtility and make function static.

Reply:
Just started to adjust the existing commit to apply/work. There's a lot more to do 🙈